Blog - create write-up for Arcus.Scripting v0.1 announement

stijnmoreels opened the pull request:
Relates to https://github.com/arcus-azure/arcus.scripting/issues/74

tomkerkhove commented on community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:
```diff
@@ -0,0 +1,76 @@
+# Announcing Arcus Scripting
```
Some general remarks:
- Looks good, but let's add some samples!

stijnmoreels replied:
Yes, done!

tomkerkhove commented on community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:
```diff
@@ -0,0 +1,76 @@
+  * [Provide script to backup an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/75)
+  * [Provide script to set the subscriptionKey on an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/39)
+* [Azure Cognitive Services](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Acognitive-services)
+  * [Provide script set to build Azure Cognitive Search & Form Recognizer](https://github.com/arcus-azure/arcus.scripting/issues/22)
```
I wouldn't include this one

mbraekman commented on community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:
```diff
@@ -0,0 +1,99 @@
+  * [Provide script to set the subscriptionKey on an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/39)
+* [Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Akey-vault)
+  * [Provide script for importing certificate as base64-string into Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues/71)
+
```
I would include the planned scripts related to Azure Logic Apps.
This way, people can see that besides offering a test framework, we'll also be providing a set  of scripts to help them during deployment.
Something like:
 _Provide scripts to enable or disable logic apps in a fixed order, preventing data-loss during an upgrade._

stijnmoreels replied:
Aha, yes, added a list item for this. Thanks!

mbraekman commented on community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:
```diff
@@ -0,0 +1,99 @@
+
+## Azure Resource Manager (ARM)
+
+The `Arcus.Scripting.ARM` PowerShell module contains the capability to inject content into a ARM template, controling the formatting of the injected content.
```
Perhaps include a bit of info as to what users will be able to use the module for in general?

```suggestion
The `Arcus.Scripting.ARM` PowerShell module has been built to help you in manipulating ARM templates before triggering a deployment, allowing you to reduce the complexity of these templates.  
Currently, this module contains the capability to inject content into an ARM template, controling the formatting of the injected content.
```

tomkerkhove commented on community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:
```diff
@@ -0,0 +1,99 @@
+
+## What's on the Horizon
+
+This summary of all the features in this first version is only the beginning of a great scripting story.
```
```suggestion
But this is only the beginning of a great scripting story!
```

tomkerkhove commented on community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:
```diff
@@ -0,0 +1,99 @@
+## What's on the Horizon
+
+This summary of all the features in this first version is only the beginning of a great scripting story.
+Here are some features that are on the horizon, but please don't heasitate to [make a suggestion](https://github.com/arcus-azure/arcus.scripting/issues/new?template=Feature_request.md) and we're happy to help where we can!
```
```suggestion
Here are some features that are on the horizon:
```

stijnmoreels replied:
No suggestions for you 😄 .

stijnmoreels replied:
Ah, nevermind, didn't see that.

tomkerkhove commented on community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:
```diff
@@ -0,0 +1,99 @@
+  * [Provide script to set the subscriptionKey on an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/39)
+* [Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Akey-vault)
+  * [Provide script for importing certificate as base64-string into Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues/71)
+
```
```suggestion

Are we missing something? Don't hesitate to [make a suggestion](https://github.com/arcus-azure/arcus.scripting/issues/new?template=Feature_request.md) and we're happy to help where we can!
```

Reason is that it's best to show what we are planning and only then ask for more suggestions; this feels more natural to me.

stijnmoreels commented:
> Some suggestions.
> 
> Would also be good to run it through https://www.grammarly.com/ as I see some spelling issues.

Grammerly doesn't work apparently in the edit window of files in GitHub. I know there are probably a lot of spelling errors. I guess that's also why this is only a draft and has to be re-read by someone.

tomkerkhove commented:
> > Some suggestions.
> > Would also be good to run it through [grammarly.com](https://www.grammarly.com/) as I see some spelling issues.
> 
> Grammerly doesn't work apparently in the edit window of files in GitHub. I know there are probably a lot of spelling errors. I guess that's also why this is only a draft and has to be re-read by someone. I'm thinking about an editorial review when this is on WordPress.
> 
> Found some errors with another tool, though and fixed them.

If you go to https://app.grammarly.com/ and open a new doc you can validate it there.

mbraekman commented on community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:
```diff
@@ -0,0 +1,103 @@
+Currently, this module contains the capability to inject content into an ARM template, controlling the formatting of the injected content.
+After importing the module, the `Inject-ArmContent` function is available to you.
+
+The [docs](https://github.com/arcus-azure/arcus.scripting/blob/master/docs/preview/features/powershell/arm.md) has an example of an OpenAPI specifications you'd want to deploy.
```
```suggestion
The [docs](https://github.com/arcus-azure/arcus.scripting/blob/master/docs/preview/features/powershell/arm.md) show you an example of injecting OpenAPI specifications into an ARM template.
```

mbraekman commented on community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:
```diff
@@ -0,0 +1,103 @@
+  * [Provide script to backup an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/75)
+  * [Provide script to set the subscriptionKey on an Azure API Management service](https://github.com/arcus-azure/arcus.scripting/issues/39)
+* [Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%3Akey-vault)
+  * [Provide script for importing certificate as base64-string into Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues/71)
```
```suggestion
  * [Provide script for importing a certificate as a base64-string into Azure Key Vault](https://github.com/arcus-azure/arcus.scripting/issues/71)
```

stijnmoreels commented:
> > > Some suggestions.
> > > Would also be good to run it through [grammarly.com](https://www.grammarly.com/) as I see some spelling issues.
> > 
> > 
> > Grammerly doesn't work apparently in the edit window of files in GitHub. I know there are probably a lot of spelling errors. I guess that's also why this is only a draft and has to be re-read by someone. I'm thinking about an editorial review when this is on WordPress.
> > Found some errors with another tool, though and fixed them.
> 
> If you go to https://app.grammarly.com/ and open a new doc you can validate it there.

OK, done, added only a missing comma.

tomkerkhove commented on community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:
```diff
@@ -0,0 +1,106 @@
+  * Provide scripts to [enable](https://github.com/arcus-azure/arcus.scripting/issues/19) or [disable](https://github.com/arcus-azure/arcus.scripting/issues/20) logic apps in a fixed order, preventing data-loss during an upgrade
+
+Are we missing something? Don't hesitate to [make a suggestion](https://github.com/arcus-azure/arcus.scripting/issues/new?template=Feature_request.md) and we're happy to help where we can!
+Thanks for reading!
```
```suggestion

Thanks for reading!
```

tomkerkhove commented on community/blog/2020-08-Announcing-Arcus-Scripting-v0.1/README.md:
```diff
@@ -0,0 +1,106 @@
+
+Are we missing something? Don't hesitate to [make a suggestion](https://github.com/arcus-azure/arcus.scripting/issues/new?template=Feature_request.md) and we're happy to help where we can!
+Thanks for reading!
+Arcus team
```
```suggestion

Arcus team
```
